lib/build.gradle.kts

```diff
@@ -17,8 +17,8 @@ dependencies {
     implementation("org.slf4j:slf4j-api:$slf4jVersion")
 
     implementation("com.fasterxml.jackson.core:jackson-databind:2.14.2")
-    api("org.glassfish.jersey.core:jersey-client:$jerseyVersion")
-    api("org.glassfish.jersey.inject:jersey-hk2:$jerseyVersion")
+//    api("org.glassfish.jersey.core:jersey-client:$jerseyVersion")
+//    api("org.glassfish.jersey.inject:jersey-hk2:$jerseyVersion")
```

Comment:
commented code

Reply:
Yep, forgot to clean up. These dependencies appear to have been unused.